.travis.yml

```
@@ -2,7 +2,7 @@ language: julia
 
 julia:
   - 1.0
-  - 1.4
+  - 1.5
```

```suggestion
  - 1
```
this shorthand will auto update when the next minor release is changed

---

A good point - thank you!